runner/browserrunner.ts

```diff
@@ -151,12 +159,10 @@ export class BrowserRunner {
   }
 
   startTest() {
-    const webserver = this.options.webserver;
-    const host = `http://${webserver.hostname}:${webserver.port}`;
-    const path = this.options.webserver.webRunnerPath;
-    const extra = (path.indexOf('?') === -1 ? '?' : '&') +
-        `cli_browser_id=${this.def.id}`;
-    this.browser.get(host + path + extra, (error) => {
+    const url = this.url;
```

Comment:
nit: not buying you much, since you only use it in one place. Either remove, or change the other `this.url`s.